File: Examples/GeodeExampleCSharp (WPF)/Extension.cs
using Geode.Extension;

namespace GeodeExampleCSharp
{
    [Module("Geode example", "Lilith", "For testing purposes only.")]
    public class Extension : GService
    {
        public MainWindow MainWindowParent;

        public Extension(MainWindow MainWindowParent)
        {
            this.MainWindowParent = MainWindowParent;
        }

        public void ShowAlert(string Message)
        {
            SendToClientAsync(In.HabboBroadcast, Message);
        }

    }
}

File: Examples/GeodeExampleVB (WinForms)/Extension.vb
Imports Geode.Extension

<[Module]("Geode example", "Lilith", "For testing purposes only.")>
Public Class Extension
    Inherits GService
    Public MainWindowParent As MainWindow

    Public Sub New(ByVal MainWindowParent As MainWindow)
        Me.MainWindowParent = MainWindowParent
    End Sub

    Public Sub ShowAlert(ByVal Message As String)
        SendToClientAsync([In].HabboBroadcast, Message)
    End Sub
End Class